WordPress/src/main/java/org/wordpress/android/ui/quickstart/QuickStartTaskState.kt

```diff
@@ -5,5 +5,6 @@ import org.wordpress.android.fluxc.store.QuickStartStore.QuickStartTask
 // represents completion state of Quick Start tasks
 data class QuickStartTaskState(
     val task: QuickStartTask,
-    val isTaskCompleted: Boolean
+    val isTaskCompleted: Boolean,
+    val isTaskShown: Boolean
```

Comment:
Ah, this one :) There is a field `isShown` in Flux-C DB layer - honestly I didn't know it's original purpose, so I left it be for the time being. But, there was one state that I needed to track on the per-site basis - when you completed a Quick Start, but have not opened the Quick Start list ("orange dot" state). I decided against adding an additional table to DB layer to track this and repurposed the `isShown` field for this (when you click on a Quick Start row I mark all tasks as `shown`, this makes sense to me) and because of this we know that all tasks were shown and we should no longer display Quick Start row. 

Now it gets a bit weird - marking all tasks as `isShown` resets the `isDone` flag (as you can see [here](https://github.com/wordpress-mobile/WordPress-FluxC-Android/blob/develop/fluxc/src/main/java/org/wordpress/android/fluxc/persistence/QuickStartSqlUtils.kt#L69) we make a new model, and the default value for `isDone` is `false`). The idea is if the task `isShown` it's also `isDone` so we can use those files interchangeably. 

Looking at this now, I should probably fix the original issue with `isDone` flag reset in Flux-C instead of taking a shortcut like this :)

Reply:
Thanks for the explanation and fix.  I reviewed and merged the FluxC pull request.  Let me know if you would like to update this pull request consequently.